woocommerce/woocommerce-android
# Introduce ReceiptCreator

Parent issue #4039 

This PR introduces ReceiptCreator. A class responsible for generating HTML receipts.

The code has been copied from woocommerce-ios repository - [ReceiptRendered](https://github.com/woocommerce/woocommerce-ios/blob/feature/stripe-terminal-sdk-integration/Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift). Most of the code has been just transformed into kotlin.

Behavioral changes (cc @ctarda @allendav @koke):
1. Amounts are always printed with two decimal places.
2. The receipt doesn't contain css classes for other CardBrands except of the one used for the payment
3. Base64 version of CardBrand icons is stored in the code not generated in runtime

Note: The unit test is very naive. However, I don't think it's worth the effort writing more granular unit tests for this behavior. Wdyt?

To test:
CI unit tests should be enough - if you want to check how the receipt looks like open "example_receipt_html.html" in a browser.

Update release notes:
